### DISCUSSION TITLE
SAM is so cool (Demo area)

### DISCUSSION THREAD

**Discussion by raymondlo84** [openvinotoolkit/openvino_notebooks#1011] [General]
Before
![bird](https://user-images.githubusercontent.com/1720147/231844663-4bb45f9a-7bcb-4e25-aa52-1543bc2e392b.jpg)
After
![download](https://user-images.githubusercontent.com/1720147/231844693-5e3be5f5-4272-4a15-b2db-385a5c2e8058.png)

**Comment by adrianboguszewski** [comment]
![sam](https://github.com/openvinotoolkit/openvino_notebooks/assets/4547501/da5d1b71-d570-45e9-89fa-87e231c8f3e3)
